src/hooks/useStripeTerminal.tsx

```diff
@@ -1069,6 +1070,10 @@ export function useStripeTerminal(props?: Props) {
     [_isInitialized, setLoading]
   );
 
+  const _getNativeSdkVersion = useCallback(async () => {
+    return await getNativeSdkVersion();
```

Comment:
should we include the `isInitialized` check here or is that redundant due to how the constants are retrieved from the SDK?

Reply:
Because it's just a const inside each sdk, we don't even need to call the terminal instance. So no matter whatever state it should return the same value.